cartography/cli.py

```diff
@@ -575,6 +583,9 @@ def main(self, argv: str) -> int:
         # TODO support parameter lookup in environment variables if not present on command line
         config: argparse.Namespace = self.parser.parse_args(argv)
         # Logging config
+        if config.logging_config:
```

Comment:
Can you add a comment linking to the Python doc for this API? This way readers can know the correct format for a logging config file.